flexmeasures/data/tests/test_annotations.py

```diff
@@ -1,13 +1,16 @@
 import pandas as pd
+from sqlalchemy import select, func
 
 from flexmeasures.data.models.annotations import Annotation, get_or_create_annotation
 from flexmeasures.data.models.data_sources import DataSource
 
 
 def test_get_or_create_annotation(db, setup_sources):
     """Save an annotation, then get_or_create a new annotation with the same contents."""
-    num_annotations_before = Annotation.query.count()
-    source = DataSource.query.first()
+    num_annotations_before = db.session.scalar(
+        select(func.count()).select_from(Annotation)
+    )
+    source = db.session.scalars(select(DataSource).limit(1)).first()
```

Comment:
limit + first

Reply:
It's a good practice to get the first item and It's suggested in the documentation.

Reply:
Ah, [I see](https://docs.sqlalchemy.org/en/20/core/connections.html#sqlalchemy.engine.CursorResult.first).